tests/components/apprise/test_notify.py

```diff
@@ -0,0 +1,68 @@
+"""The tests for the apprise notification platform."""
+from unittest.mock import patch
+
+from homeassistant.setup import async_setup_component
+
+BASE_COMPONENT = "notify"
+
+
+async def test_apprise_config_load_fail(hass):
+    """Test apprise configuration failures."""
+
+    config = {"apprise": {"name": "test", "platform": "apprise", "config": "/path/"}}
+
+    with patch("apprise.AppriseConfig.add", return_value=False):
+        assert await async_setup_component(hass, BASE_COMPONENT, config)
+        await hass.async_block_till_done()
+
```

Comment:
That's the grey area. Apprise does all of the grunt work that normally otherwise sits in your `component/*` directories.  There isn't much to test since it's [already handle](https://codecov.io/github/caronc/apprise).  The tests I wrote just test the responses and attempt to access all portions of the new code i added to Home Assistant.  I can see with the merge i did last night i missed one.  I'll revisit that soon and do another push.

Reply:
I think we can check if the service for this platform has been registered.

Reply:
Done!